src/app/Http/Controllers/CrudFeatures/Reorder.php

```diff
@@ -21,6 +26,7 @@ public function reorder()
 
         // get all results for that entity
         $this->data['entries'] = $this->crud->getEntries();
+        $this->data['entries'] = $this->data['entries']->filter($this->reorder_filter_callback);
```

Comment:
...what is that's `null`?

Reply:
nothing happens

The function filter already manages this case:
/vendor/laravel/framework/src/Illuminate/Support/Collection.php
```php
    /**
     * Run a filter over each of the items.
     *
     * @param  callable|null  $callback
     * @return static
     */
    public function filter(callable $callback = null)
    {
        if ($callback) {
            return new static(Arr::where($this->items, $callback));
        }

        return new static(array_filter($this->items));
    }
```

Anyway we could add an extra verification just in case this filter function changes:
```php
        if(is_callable($this->reorder_filter_callback)) {
            $this->data['entries'] = $this->data['entries']->filter($this->reorder_filter_callback);
        }
```

Reply:
It might be preferable to ensure that it is callable especially since it is settable by a derived class (although this case falls into `the developer shouldn't do silly things`).

Reply:
It's done!